src/plugins/bfetch/public/streaming/from_streaming_xhr.test.ts

```diff
@@ -182,7 +182,7 @@ test('does not emit when gets error response', () => {
   expect(error).toHaveBeenCalledTimes(1);
   expect(error.mock.calls[0][0]).toBeInstanceOf(Error);
   expect(error.mock.calls[0][0].message).toMatchInlineSnapshot(
-    `"Batch request failed with status 400"`
+    `"Error connecting to server - code 400"`
```

Comment:
Where does this message appear?  Is it user facing?

Reply:
Its the same toast error as shown in https://github.com/elastic/kibana/issues/145810 but with a different error code. The meaning of the error is a bit different. The other error changed in this PR is when the browser loses connection to kibana due to a network interruption. This message is when either kibana or ES returns an error code.

Reply:
I placed a description in the PR description. You can see where the message is displayed in both the toast and in the histogram error.